types/googlemaps/reference/marker.d.ts

```diff
@@ -458,6 +458,17 @@ declare namespace google.maps {
      * @see {@link https://developers.google.com/maps/documentation/javascript/reference/marker#MarkerLabel Maps JavaScript API}
      */
     interface MarkerLabel {
+        /**
+         * The className property of the label's element (equivalent to the element's class attribute).
+         * Multiple space-separated CSS classes can be added. Default is no CSS class (an empty string).
+         * The font color, size, weight, and family can only be set via the other properties of MarkerLabel.
```

Comment:
I suggest you mark the mention of `MarkerLabel` as a link to it.

```suggestion
         * The font color, size, weight, and family can only be set via the other properties of {@link MarkerLabel}.
```